plugins/aladino/actions/merge.go

```diff
@@ -23,6 +24,11 @@ func Merge() *aladino.BuiltInAction {
 func mergeCode(e aladino.Env, args []aladino.Value) error {
 	t := e.GetTarget().(*target.PullRequestTarget)
 
+	if t.PullRequest.GetState() != "open" || t.PullRequest.GetDraft() {
+		log.Println("merge: skipping action because pull request is not open or is a draft")
+		return nil
+	}
```

Comment:
I would suggest splitting this condition into two.
```suggestion
	if t.PullRequest.GetDraft() {
		log.Println("merge: skipping action because pull request is on draft")
		return nil
	}

	if t.PullRequest.GetState() != "open" {
		log.Println("merge: skipping action because pull request is not open")
		return nil
	}
```

WDYT?

Reply:
Lets leave this as is.